crd/fleet/fleet-crds.yaml

```diff
@@ -4064,6 +5652,40 @@ spec:
                   - resource
                   type: object
                 type: array
+              endTime:
+                description: EndTime is the time failover reached its final state.
+                format: date-time
+                type: string
+              internal:
+                description: Internal is used by the system controllers. You should
```

Comment:
Replace "You should not directly depend on..." with "Do not rely on the information in this section"